src/DssSpell.t.sol

```diff
@@ -563,4 +563,123 @@ contract DssSpellTest is DssSpellTestBase {
+        TeleportRouterLike router = TeleportRouterLike(addr.addr("MCD_ROUTER_TELEPORT_FW_A"));
+
+        bytes32 ilk = "TELEPORT-FW-A";
+        bytes23 domain = "ETH-MAIN-A";
+
+        // Sanity checks
+        assertEq(vat.wards(address(join)), 1);
+
+        assertEq(join.wards(address(oracleAuth)), 1);
+        assertEq(join.wards(address(router)), 1);
+        assertEq(join.wards(address(esm)), 1);
+        assertEq(join.vow(), address(vow));
+        assertEq(join.daiJoin(), address(daiJoin));
+        assertEq(join.vat(), address(vat));
+        assertEq(join.ilk(), ilk);
+        assertEq(join.domain(), domain);
+
+        assertEq(oracleAuth.signers(0xaC8519b3495d8A3E3E44c041521cF7aC3f8F63B3), 1);
+        assertEq(oracleAuth.signers(0x4f95d9B4D842B2E2B1d1AC3f2Cf548B93Fd77c67), 1);
+        assertEq(oracleAuth.signers(0xE6367a7Da2b20ecB94A25Ef06F3b551baB2682e6), 1);
+        assertEq(oracleAuth.signers(0xFbaF3a7eB4Ec2962bd1847687E56aAEE855F5D00), 1);
+        assertEq(oracleAuth.signers(0x16655369Eb59F3e1cAFBCfAC6D3Dd4001328f747), 1);
+        assertEq(oracleAuth.signers(0xC9508E9E3Ccf319F5333A5B8c825418ABeC688BA), 1);
+        assertEq(oracleAuth.signers(0xA8EB82456ed9bAE55841529888cDE9152468635A), 1);
+        assertEq(oracleAuth.signers(0x83e23C207a67a9f9cB680ce84869B91473403e7d), 1);
+        assertEq(oracleAuth.signers(0xDA1d2961Da837891f43235FddF66BAD26f41368b), 1);
+        assertEq(oracleAuth.signers(0x4b0E327C08e23dD08cb87Ec994915a5375619aa2), 1);
+        assertEq(oracleAuth.signers(0xfeEd00AA3F0845AFE52Df9ECFE372549B74C69D2), 1);
+        assertEq(oracleAuth.signers(0x8aFBD9c3D794eD8DF903b3468f4c4Ea85be953FB), 1);
+        assertEq(oracleAuth.signers(0x8de9c5F1AC1D4d02bbfC25fD178f5DAA4D5B26dC), 1);
+        assertEq(oracleAuth.signers(0xd94BBe83b4a68940839cD151478852d16B3eF891), 1);
+        assertEq(oracleAuth.signers(0xa580BBCB1Cee2BCec4De2Ea870D20a12A964819e), 1);
+        assertEq(oracleAuth.signers(0x75ef8432566A79C86BBF207A47df3963B8Cf0753), 1);
+        assertEq(oracleAuth.signers(0xD27Fa2361bC2CfB9A591fb289244C538E190684B), 1);
+        assertEq(oracleAuth.signers(0x60da93D9903cb7d3eD450D4F81D402f7C4F71dd9), 1);
+        assertEq(oracleAuth.signers(0x71eCFF5261bAA115dcB1D9335c88678324b8A987), 1);
+        assertEq(oracleAuth.signers(0x77EB6CF8d732fe4D92c427fCdd83142DB3B742f7), 1);
+        assertEq(oracleAuth.signers(0x8ff6a38A1CD6a42cAac45F08eB0c802253f68dfD), 1);
+        assertEq(oracleAuth.signers(0x130431b4560Cd1d74A990AE86C337a33171FF3c6), 1);
+        assertEq(oracleAuth.signers(0x3CB645a8f10Fb7B0721eaBaE958F77a878441Cb9), 1);
+        assertEq(oracleAuth.signers(0xd72BA9402E9f3Ff01959D6c841DDD13615FFff42), 1);
+
+        assertEq(oracleAuth.teleportJoin(), address(join));
+        assertEq(oracleAuth.threshold(), 13);
+        assertEq(oracleAuth.wards(address(esm)), 1);
+
+        assertEq(router.gateways(domain), address(join));
+        assertEq(router.domains(address(join)), domain);
```

Comment:
We should be checking the other two domains as well.

Reply:
This is done in the `checkTeleportFWIntegration`

Reply:
Ah right!